join_request/josep.yml

```
@@ -0,0 +1,52 @@
+collective: Potoc
+member:
+  name: Josep M Sobrepere
+  address: 15roJ4ZrgrZam5BQWJgiGHpgp7ShFQBRNLq6qUfiNqXDZjMK
+  github: josepot
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 15roJ4ZrgrZam5BQWJgiGHpgp7ShFQBRNLq6qUfiNqXDZjMK
  matrix:
  github: josepot
```